ui/src/store/resource.ts

```diff
@@ -109,8 +111,9 @@ export const ResourceStore = types
     versions: types.map(Version),
     catalog: types.optional(types.map(Catalog), {}),
     kinds: types.optional(KindStore, {}),
+    tags: types.optional(TagStore, {}),
     sortBy: types.optional(types.enumeration(Object.values(SortByFields)), SortByFields.Unknown),
-    tags: types.optional(types.map(Tag), {}),
+    allcategories: types.optional(types.map(Category), {}),
```

Comment:
Could you please change this to just `categories`

Reply:
```suggestion
categories: types.optional(types.map(Category), {}),
```

Reply:
we have already a view with same name`categories`